src/manifolds/ConnectionManifold.jl

```diff
@@ -162,13 +197,13 @@ in an embedded space.
 exp(::AbstractConnectionManifold, ::Any...)
 
 @decorator_transparent_fallback function exp!(M::AbstractConnectionManifold, q, p, X)
-    tspan = (0.0, 1.0)
-    A = get_default_atlas(M)
-    i = get_chart_index(M, A, p)
-    B = induced_basis(M, A, i, TangentSpace)
-    sol = solve_exp_ode(M, p, X, tspan, B; dense=false, saveat=[1.0])
-    n = length(p)
-    return copyto!(q, sol.u[1][(n + 1):end])
+    return retract!(
+        M,
+        q,
+        p,
+        X,
+        ODEXxponentialRetration(ManifoldsBase.default_retraction_method(M)),
```

Comment:
What do you think about leaving `exp!` unimplemented and instead returning `ODEExponentialRetration` as the default retraction?

Reply:
Thats also a neat idea :)

Reply:
Thinking about it, maybe it is not the best idea to make that the default retraction, because when someone _defines_ `exp!` for his/her manifold, we actually want that to be the default.

Reply:
Is it ok to keep `ExponentialRetraction` as default? I think if exp! is present we want to use that instead of solving the ODE, so I would like to keep `ExponentialRetration` as a default retraction,

Reply:
That's true. So it really makes sense to keep it this way.